dusty/command_file.py

```diff
@@ -0,0 +1,129 @@
+import os
+from nose.tools import nottest
+
+from . import constants
+from .source import Repo
+from .compiler.compose.helper_functions import container_code_path
+from .systems.docker.helper_functions import spec_for_service
```

Comment:
Having a top-level module go this deep into the imports seems like it may blow up later, should the helper functions be higher up somewhere?